### ISSUE TITLE
Insensitive Touchpad

### ISSUE THREAD

**Issue by allegro98** [Microsoft/vscode#35521]
- VSCode Version: Code 1.16.1 (27492b6bf3acb0775d82d2f87b25a93490673c6d, 2017-09-14T16:38:23.027Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
python|don|0.7.0
latex-workshop|Jam|3.5.4

---

Steps to Reproduce:

1.open the vs code in a xps 15,enlarge the screen at 125%
2.move your finger.

Reproduces without extensions: Yes/No  yes

when I change the screen at 100%,and set as 125% again,I use the touchpad move document as sensitive as Microsoft Edge. But when I restart my computer, this problem happen as usual.

**Comment by chrmarti** [comment]
How do you enlarge the screen? Is that the magnifier in Windows 10? Are you using a keyboard shortcut to do that?

**Comment by allegro98** [comment]
Thank you for your help!
My computer using Chinese, so I do not know how to translate it......
My computer is Dell xps 15, and run Windows 10, as a result of high resolution screen, Windows 10 have enlarge screen for convenient using. And I think it made this bug.
For example,I moved my finger to slide some codes, but 3s later, the code start sliding, it is hard to use Touchpad. When I use mouse, it slide smoothly, so it is a compatibility problem.
And I found the problem could be fixed temporary, I changed the rate in setting, like this,125% to 100% to  125%.
I think vs code lead to the problem.
Thanks!
